packages/test/test-end-to-end-tests/src/test/gc/gcUnreferencedTimestamp.spec.ts

```diff
@@ -523,7 +541,12 @@ describeCompat("GC unreferenced timestamp", "NoCompat", (getTestObjectProvider)
 			 * observed by summarizer. So, the summarizer does not see this reference directly but only when B is realized.
 			 */
 			it(`Scenario 6 - Reference added via new unreferenced nodes and removed`, async () => {
-				const { summarizer } = await createSummarizer(provider, mainContainer);
+				// Disable new Reference Detection behavior for now
+				settings[detectOutboundRoutesViaDDSKey] = true;
```

Comment:
Let's create work items for these 2 scenarios and schedule it sometime in the next couple of sprints.

Reply:
They're the same root cause, same as it already tracked by AB#5757. I updated that bug's description to explain this case as well (it just depends on whether a given node is referenced or unreferenced by the time GC runs)